src/components/TodoComponents/SearchForm.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Todo.css'
+
+class SearchForm extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            searchKey: ''
+        }
+    }
+
+    handleInput = (event)=>{
+        this.setState({
+            searchKey: event.target.value
+        })
+    }
+
+    hideTask = (event)=>{
```

Comment:
Why are these on here twice? You can pass them from App.js and pass a parameter from this component's state into the props method then you would just need an input handler and searchKey state.